CodedEvents/backend/models.py

```diff
@@ -1,3 +1,50 @@
 from django.db import models
+from django.contrib.auth.models import User
```

Comment:
A little description on each method will be helpful. Helps in future cases if we want to add a front-end view to the api like swagger etc.

Reply:
Yea but there aren't any methods yet though?

Reply:
Yeah wrong wording, we still need a little description/commenting for the models though